Duplicati/Server/WebServer/AuthenticationHandler.cs

```diff
@@ -179,14 +179,19 @@ public override bool Process(HttpServer.IHttpRequest request, HttpServer.IHttpRe
                         return true;
                     }
 
+                    var password = Program.DataConnection.ApplicationSettings.WebserverPassword;
+
+                    if (request.Headers["User-Agent"] != null && request.Headers["User-Agent"].StartsWith("Duplicati TrayIcon Monitor"))

```

Comment:
Yeah X-TrayIcon-Login could be option. Logic was that if client is not browser (user), but TrayIcon it will not check FormPassword, but TrayIconPassword.